test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ESClientYamlSuiteTestCase.java

```diff
@@ -308,6 +289,25 @@ private static Version readVersionsFromInfo(RestClient restClient, int numHosts)
     }
 
     public void test() throws IOException {
+        //skip test if it matches one of the blacklist globs
+        for (BlacklistedPathPatternMatcher blacklistedPathMatcher : blacklistPathMatchers) {
+            String testPath = testCandidate.getSuitePath() + "/" + testCandidate.getTestSection().getName();
+            assumeFalse("[" + testCandidate.getTestPath() + "] skipped, reason: blacklisted", blacklistedPathMatcher
+                .isSuffixMatch(testPath));
+        }
+
+        restTestExecutionContext.clear();
```

Comment:
Sure, I moved it back.  I guess the original reasoning for putting it after the blacklist check could have been that it would save some processing when there are many adjacent blacklisted tests, but the amount of processing saved is tiny because clearing an already cleared context does very little.